pkg/sql/parser/testdata/show_completions

```diff
@@ -0,0 +1,47 @@
+parse
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat'
+----
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat'
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- fully parenthesized
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- literals removed
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- identifiers removed
+
+parse
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat'
+----
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat'
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- fully parenthesized
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- literals removed
+SHOW COMPLETIONS AT OFFSET 1 FOR 'creat' -- identifiers removed
+
+parse
+SHOW COMPLETIONS AT OFFSET 1 FOR 'CREAT'
+----
+SHOW COMPLETIONS AT OFFSET 1 FOR 'CREAT'
+SHOW COMPLETIONS AT OFFSET 1 FOR 'CREAT' -- fully parenthesized
+SHOW COMPLETIONS AT OFFSET 1 FOR 'CREAT' -- literals removed
+SHOW COMPLETIONS AT OFFSET 1 FOR 'CREAT' -- identifiers removed
+
+parse
+SHOW COMPLETIONS AT OFFSET 7 FOR 'SELECT 1'
+----
+SHOW COMPLETIONS AT OFFSET 7 FOR 'SELECT 1'
+SHOW COMPLETIONS AT OFFSET 7 FOR 'SELECT 1' -- fully parenthesized
+SHOW COMPLETIONS AT OFFSET 7 FOR 'SELECT 1' -- literals removed
+SHOW COMPLETIONS AT OFFSET 7 FOR 'SELECT 1' -- identifiers removed
+
+parse
+show completions at offset 10 for 'select * fro'
+----
+SHOW COMPLETIONS AT OFFSET 10 FOR 'select * fro' -- normalized!
+SHOW COMPLETIONS AT OFFSET 10 FOR 'select * fro' -- fully parenthesized
+SHOW COMPLETIONS AT OFFSET 10 FOR 'select * fro' -- literals removed
```

Comment:
hm, i would have expected "literals removed" to remove the parts in single quotes. do you know why not?

Reply:
Yeah I didn't update the formatter to remove literals, fixed.